sql/core/src/test/resources/sql-tests/results/change-column.sql.out

```diff
@@ -195,7 +195,7 @@ ALTER TABLE temp_view CHANGE a TYPE INT COMMENT 'this is column a'
 struct<>
 -- !query 20 output
 org.apache.spark.sql.AnalysisException
-temp_view is a temp view not a table.; line 1 pos 0
+Cannot alter a view with ALTER TABLE. Please use ALTER VIEW instead; line 1 pos 0
```

Comment:
Hi, @cloud-fan and @imback82 
`Cannot alter a view with ALTER TABLE.` looks okay, but `Please use ALTER VIEW instead` looks like we mislead the users to try `ALTER VIEW temp_view CHANGE a TYPE INT COMMENT 'this is column a'`.

Reply:
The other changes are the same. The directional message parts look misleading.

Reply:
This is a good point. How about just `Cannot alter a temp view with ALTER TABLE.` as the error message?

Reply:
+1 for @cloud-fan 's advice.